lib/cavendish/assets/app.config.ts.erb

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable no-undef */
+import { ExpoConfig, ConfigContext } from 'expo/config'; 
+
+const { APP_ENV } = process.env;
+
+type AppEnvironment = 'development' | 'staging' | 'production';
+
+const appEnv: AppEnvironment = APP_ENV as AppEnvironment || 'development';
```

Comment:
I think that the const type declaration is not necessary, because `'development'` is one of the possible values of `AppEnvironment` and you're declaring that `APP_ENV` is also of that type.

I think that:

```suggestion
const appEnv = APP_ENV as AppEnvironment || 'development';
```

Should suffice

Reply:
Deleted the redundant type :+1: